schema/crdb/dbinit.sql

```diff
@@ -2399,6 +2399,39 @@ CREATE TABLE IF NOT EXISTS omicron.public.tuf_repo_artifact (
 
 /*******************************************************************/
 
+-- The source of the software release that should be deployed to the rack.
+CREATE TYPE IF NOT EXISTS omicron.public.target_release_source AS ENUM (
+    'install_dataset',
+    'system_version'
+);
+
+-- The software release that should be deployed to the rack.
+CREATE TABLE IF NOT EXISTS omicron.public.target_release (
+    generation INT8 NOT NULL PRIMARY KEY,
```

Comment:
I'd be inclined to call this `version`.  That's what we used in `bp_target`.  But we do use both `version` and `generation` in different places and I'm not sure really sure I could explain when to use which.  Either seems okay.

Reply:
I think `generation` here is both more precise and clearer, since it's less likely to be mixed up with a semantic version.